package-lock.json

```diff
@@ -2611,6 +2642,11 @@
         }
       }
     },
+    "coffee-script": {
+      "version": "1.10.0",
+      "resolved": "https://registry.npmjs.org/coffee-script/-/coffee-script-1.10.0.tgz",
+      "integrity": "sha1-EpOLz5vhlI+gBvkuDEyegXBRCMA="
+    },
```

Comment:
I'm a bit worried about CoffeeScript being included as a non-dev dependency. This is apparently as a result of FHIRPath being used, although I'm not sure why, as FHIRPath is not written in CoffeeScript.